vendor/brave-ios/Ledger/Data/BATLedgerDatabase.mm

```diff
@@ -12,6 +12,7 @@
 
 #define BLOG(__severity) RewardsLogStream(__FILE__, __LINE__, __severity).stream()
 
+// TO BE REMOVED:
```

Comment:
Prefix `TODO:`

Reply:
this will be handled by different PR by @kylehickinson